### DISCUSSION TITLE
[keyball39]Qの横一列のkeyを押すと文字が多重入力される

### DISCUSSION THREAD

**Discussion by ken0385** [Yowkees/keyball#469] [Q&A]
左基盤、上列のQWERTの各Keyを押すと複数の文字が以下のように表示されます。
qを押すと qaz
wを押すとwsx
eを押すとedc
と縦一列の文字が順番に並びます。

左、右どちらにUSB-Cを指しても同じ状況です。
半田を触り直しましたが改善されません。

なにかアドバイスお願いできますか？

![IMG_0757](https://github.com/Yowkees/keyball/assets/155802829/25db7e88-668a-4ecb-92d9-9710805c3f07)

**Comment by apio1969** [comment]
今回の原因に直接関係するかどうかはわかりませんが、
写真を拝見する限り、キーソケットのハンダ付けの向き（？）が反対だと思います。
キーソケットの向きは、基板に黒くプリントされているように設置するのが正しいソケットのは位置です。
ビルドガイドの
3-3．キーソケットのハンダ付けを一度確認してみてはいかがでしょうか。
キーソケットを正しい向きにハンダ付けすることで改善する可能性が高いように思います。
写真を見た限りのあくまで素人判断です。改善されなかったらすみません。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
また、apio1969様もご回答ありがとうございました。

お写真をみると、下記写真の緑丸部分が半田屑でショートしているように見えます。この状態ですと、ROW0線が常にGNDに接続されてしまうので、この症状が起こるように思います。
ここを半田ごてで除去してみてください。

また、確かにキーソケットが逆に半田付けされています。このままでも使用できると思いますが、キーソケットの開口幅が左右穴で微妙に違うようで、将来的にキーが効かなくなる等の不具合が出るかも知れません。
そのため、やり直すのが無難かと思います。１本は安価なもので良いので半田ごてが２本あると簡単に取り外すことができます。

以上、お手数ですがご対応お願いいたします。

![KB39_qazfix](https://github.com/Yowkees/keyball/assets/84209772/3f649541-3190-409b-906c-3ad857e653f7)

**Comment by ken0385** [comment]
apio1969様、Yowkees様
ありがとうございます。
両方修正させていただき無事動くようになりました。

**Comment by Yowkees** [comment]
ご自身で修正していただき、無事に動くようになったようで安心しました！
ぜひ末永くご愛用いただければ幸いです。今後ともどうぞよろしくお願いいたします。
